nservicebus/upgrades/5to6.md

```diff
@@ -600,4 +600,17 @@ snippet: 5to6delayed-delivery
 
 ## Serializers
 
-The BSON serializer build into the core has been removed. Use the [Newtonsoft serializer](/nservicebus/serialization/newtonsoft.md) as a replacement. Also see the [Newtonsoft BSON sample](/samples/serializers/newtonsoft-bson/).
\ No newline at end of file
+The BSON serializer built into the core has been removed. Use the [Newtonsoft serializer](/nservicebus/serialization/newtonsoft.md) as a replacement. Also see the [Newtonsoft BSON sample](/samples/serializers/newtonsoft-bson/).
+
+## Dependency injection
+With we no longer support property injection via the built-in container and recommend that you switch to constructor injection where applicable. Should you still want to use property injection you can achieve this by using one of our supported external containers and configure property injection using the native configuration API provided by that container.
+
+As a result of this `ConfigureProperty<T>()` is no longer provided on our `IConfigureComponents` abstraction. Also note that with the removal of property injection `IConfigureComponents` metods no longer return `IComponentConfig` since that interface was only used to provide property injection.
+
+### Setting property on handlers handlers
+Version 5 allowed you to explicitly set handler properties via the `.InitializeHandlerProperty<THandler>("propertyName",value)` method. This is no longer available in version 6 and we recommend you to extract those properties into a "settings" class that you could inject via normal constructor injection into your handlers.
+
+### Implementing your own container adapter
+Since property injection is no longer required property injection can optionally be supported should the underlying container support it.
+
+Practically this means that the container test suite no longer contains tests to check property injection and `ConfigureProperty` on the `IContainer` abstraction has been removed.
```

Comment:
working on updating the containers. I'll get back to this one once I've done a few since that might help me phrase this one better. 

https://media.giphy.com/media/JDKxRN0Bvmm2c/giphy.gif